packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -607,3 +608,176 @@ class _AnimatedDefaultTextStyleState extends AnimatedWidgetBaseState<AnimatedDef
     );
   }
 }
+
+/// Specifies which of the children to show. See [AnimatedCrossFade].
+///
+/// The child that is shown will fade in, and while the other will fade out.
+enum CrossFadeState {
+  /// A state where the first child is shown and the second child is completely
```

Comment:
"A state where" is redundant. I'd just say "Show the first child and hide the second." or some such.